[AUTOCUT] OS Distribution Build Failed for ml-commons-2.6.0

***Received Error***: **Error building ml-commons, retry with: ./build.sh manifests/2.6.0/opensearch-2.6.0.yml --component ml-commons**.
                      The distribution build for ml-commons has failed.
                      Please see build log at https://build.ci.opensearch.org/job/distribution-build-opensearch/7102/consoleFull

See such error
```
/tmp/tmpzb01ayqh/ml-commons/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LogisticRegression.java:107: error: unmappable character (0xE2) for encoding US-ASCII
        // loggingInterval ??? Log the loss after this many iterations. If -1 don't log anything.
                           ^
/tmp/tmpzb01ayqh/ml-commons/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LogisticRegression.java:107: error: unmappable character (0x80) for encoding US-ASCII
        // loggingInterval ??? Log the loss after this many iterations. If -1 don't log anything.
                            ^
/tmp/tmpzb01ayqh/ml-commons/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LogisticRegression.java:107: error: unmappable character (0x93) for encoding US-ASCII
        // loggingInterval ??? Log the loss after this many iterations. If -1 don't log anything.
```
```
> Task :opensearch-ml-plugin:javadoc FAILED
/tmp/tmp5a4so80d/ml-commons/plugin/build/generated/sources/delombok/java/main/org/opensearch/ml/model/MLModelAutoReloader.java:69: error: unmappable character (0xEF) for encoding US-ASCII
     * constructor method??? init all the params necessary for model auto reloading
                         ^
/tmp/tmp5a4so80d/ml-commons/plugin/build/generated/sources/delombok/java/main/org/opensearch/ml/model/MLModelAutoReloader.java:69: error: unmappable character (0xBC) for encoding US-ASCII
     * constructor method??? init all the params necessary for model auto reloading
                          ^
/tmp/tmp5a4so80d/ml-commons/plugin/build/generated/sources/delombok/java/main/org/opensearch/ml/model/MLModelAutoReloader.java:69: error: unmappable character (0x8C) for encoding US-ASCII
     * constructor method??? init all the params necessary for model auto reloading
                           ^
3 errors
```

--- a/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LogisticRegression.java
+++ b/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LogisticRegression.java
@@ -104,7 +104,7 @@ public class LogisticRegression implements Trainable, Predictable {
             throw new IllegalArgumentException("MiniBatchSize should not be negative.");
         }
 
-        // loggingInterval – Log the loss after this many iterations. If -1 don't log anything.
+        // loggingInterval - Log the loss after this many iterations. If -1 don't log anything.
         if (parameters.getLoggingInterval() != null && parameters.getLoggingInterval() < -1) {
             throw new IllegalArgumentException("Invalid Logging intervals");
         }

